src/Sulu/Bundle/ContactBundle/Entity/Address.php

```diff
@@ -355,13 +356,29 @@ public function setCountry(\Sulu\Bundle\ContactBundle\Entity\Country $country =
     /**
      * Get country.
      *
-     * @return \Sulu\Bundle\ContactBundle\Entity\Country
+     * @return string
      */
     public function getCountry()
     {
         return $this->country;
     }
 
+    /**
+     * Get country name.
+     *
+     * @param string $displayLocale
+     *
+     * @return string|null
+     */
+    public function getCountryName($displayLocale = null)
+    {
+        if (!$this->country) {
+            return null;
+        }
+
+        return Intl::getRegionBundle()->getCountryName($this->country, $displayLocale);
```

Comment:
Why do we have a `CountryRepository`, and only use it sometimes? I am totally ok with the line here, but then I am wondering why we are not accessing it this way everywhere.

Reply:
I did keep the CountryRepository to keep the old UI running. I think in this case it would be better to deprecate the CountryRepository and remove it with 2.0?